ui/src/app/cohort-search/modifier-page/modifier-page.component.html

```diff
@@ -1,7 +1,7 @@
 <form [formGroup]="form">
   <section class="form-block">
     <div class="modifier-header">
-      The Following Modifiers are Optional
+      The Following Modifiers are Optional and Apply to All Selected Criteria
```

Comment:
Can we make this sentence have lower case for all the words?

Reply:
Done